internal/server/server.go

```diff
@@ -87,10 +90,35 @@ func (s *Server) Exec() {
 	api.GET("/logs/:namespace/:layer/:run/:attempt", s.API.GetLogsHandler)
 	api.GET("/run/:namespace/:layer/:run/attempts", s.API.GetAttemptsHandler)
 
+	// start a goroutine to refresh webhook handlers every minute
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	go s.refreshWebhookHandlers(ctx)
+
 	e.Logger.Fatal(e.Start(s.config.Server.Addr))
 	log.Infof("burrito server started on addr %s", s.config.Server.Addr)
 }
 
 func handleHealthz(c echo.Context) error {
 	return c.String(http.StatusOK, "OK")
 }
+
+func (s *Server) refreshWebhookHandlers(ctx context.Context) {
```

Comment:
So this refreshes the webhook configuration every 5 minutes in case of the user has configured a new repository?

Reply:
Yes, the timing choice is a bit random, I will make it customizable in the config